pilot/cmd/pilot-agent/main.go

```diff
@@ -75,6 +75,9 @@ var (
 	disableInternalTelemetry bool
 	loggingOptions           = log.DefaultOptions()
 
+	// pilot agent config
+	appHTTPProbers string
```

Comment:
could we make these kind of names consistent?

Reply:
Done. Changed all to `kubeAppHTTPProbers`.